config/defines.inc.php

```diff
@@ -29,13 +29,14 @@
     define('_PS_MODE_DEV_', true);
 }
 
-/* Compatibility warning */
-    define('_PS_DISPLAY_COMPATIBILITY_WARNING_', false);
```

Comment:
Maybe this line must be stay here, what's append if this constant is undefined ?